src/Protocol/Diaspora.php

```diff
@@ -470,24 +474,24 @@ public static function dispatchPublic($msg, bool $fetched = false)
 		}
 
 		$importer = ["uid" => 0, "page-flags" => User::PAGE_FLAGS_FREELOVE];
-		$success = self::dispatch($importer, $msg, $fields, $fetched);
+		$success = self::dispatch($importer, $msg, $fields, $direction);
 
 		return $success;
 	}
 
 	/**
 	 * Dispatches the different message types to the different functions
 	 *
-	 * @param array            $importer Array of the importer user
-	 * @param array            $msg      The post that will be dispatched
-	 * @param SimpleXMLElement $fields   SimpleXML object that contains the message
-	 * @param bool             $fetched  The message had been fetched (default "false")
+	 * @param array            $importer  Array of the importer user
+	 * @param array            $msg       The post that will be dispatched
+	 * @param SimpleXMLElement $fields    SimpleXML object that contains the message
+	 * @param int              $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.

Reply:
How do I do this the best?

Reply:
You can add something like this: `One of: self::FETCHED, self::PUSHED or self::FORCE_PUSHED`